.storybook/main.cjs

```diff
@@ -0,0 +1,26 @@
+const { typescript: svelteTS } = require('svelte-preprocess');
```

Comment:
These files are used to configure storybook. I expect these configs will not change a lot during dev time